File: src/Model/Entities/Torn.java
package hospiolot.model.business.entities;

import hospiolot.model.business.entities.Entity;


public class Torn extends Entity {

    private String modalitat;

    public Torn(String modalitat) {
        this.modalitat = modalitat;
    }

    public String getModalitat() {
        return modalitat;
    }

    public void setModalitat(String modalitat) {
        this.modalitat = modalitat;
    }

    @Override
    public String toString() {
        return  modalitat ;
    }

}

File: BackEnd/src/Model/Entities/Torn.ts
import Entity from "./Entity";

class Torn{

    private modalitat:string;

    public constructor(modalitat:string) {
        this.modalitat = modalitat;
    }

    public getModalitat():string {
        return this.modalitat;
    }

    public setModalitat(modalitat:string):void {
        this.modalitat = modalitat;
    }

    public toString() : string {
        return this.modalitat; 
    }
    
}

export = Torn;